models/prices/prices_stability_rank.sql

```diff
@@ -0,0 +1,129 @@
+             ('avalanche_c', 'WETH.e', 0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab, 6),
+             ('avalanche_c', 'BTC.b', 0x152b9d0fdc40c096757f570a51e494bd4b943e50, 7),
+             ('avalanche_c', 'JOE', 0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd, 8),
+             ('avalanche_c', 'MIM', 0x130966628846bfd36ff31a822705796e8cb8c18d, 9),
+             ('avalanche_c', 'DAI.e', 0xd586e7f844cea2f87f50152665bcbc2c279d8d70, 10),
+             ('base', 'USDbC', 0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca, 1),
+             ('base', 'WETH', 0x4200000000000000000000000000000000000006, 2),
+             ('base', 'USDC', 0x833589fcd6edb6e08f4c7c32d4f71b54bda02913, 3),
+             ('base', 'DAI', 0x50c5725949a6f0c72e6c4a641f24049a917db0cb, 4),
+             ('base', 'axlUSDC', 0xeb466342c4d449bc9f53a865d5cb90586f405215, 5),
+             ('base', 'AERO', 0x940181a94a35a4569e4529a3cdfb74e38fd98631, 6),
+             ('base', 'cbETH', 0x2ae3f1ec7f1f5012cfeab0185bfc7aa3cf0dec22, 7),
+             ('base', 'wstETH', 0xc1cba3fcea344f92d9239c08c0568f6f2f0ee452, 8),
+             ('base', 'BALD', 0x27d2decb4bfc9c76f0309b8e88dec3a601fe25a8, 9),
+             ('base', 'MIM', 0x4a3a6dd60a34bb2aba60d73b4c88315e9ceb6a3d, 10),
+             ('bnb', 'USDT', 0x55d398326f99059ff775485246999027b3197955, 1),
+             ('bnb', 'WBNB', 0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c, 2),
+             ('bnb', 'BUSD', 0xe9e7cea3dedca5984780bafc599bd69add087d56, 3),
+             ('bnb', 'USDC', 0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d, 4),
+             ('bnb', 'ETH', 0x2170ed0880ac9a755fd29b2688956bd959f933f8, 5),
+             ('bnb', 'BTCB', 0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c, 6),
+             ('bnb', 'CAKE', 0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82, 7),
+             ('bnb', 'DAI', 0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3, 8),
+             ('bnb', 'HAY', 0x0782b6d8c4551b9760e74c0545a9bcd90bdc41e5, 9),
+             ('bnb', 'XRP', 0x1d2f0da169ceb9fc7b3144628db156f3f6c60dbe, 10),
+             ('celo', 'cUSD', 0x765de816845861e75a25fca122bb6898b8b1282a, 1),
+             ('celo', 'cEUR', 0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73, 2),
+             ('celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 3),
+             ('celo', 'BTC', 0xd629eb00deced2a080b7ec630ef6ac117e614f1b, 4),
+             ('celo', 'BIFI', 0x639a647fbe20b6c8ac19e48e2de44ea792c62c5c, 5),
+             ('celo', '', 0x23ee2343b892b1bb63503a4fabc840e0e2c6810f, 6),
+             ('ethereum', 'WETH', 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2, 1),
+             ('ethereum', 'USDT', 0xdac17f958d2ee523a2206206994597c13d831ec7, 2),
+             ('ethereum', 'USDC', 0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48, 3),
+             ('ethereum', 'DAI', 0x6b175474e89094c44da98b954eedeac495271d0f, 4),
+             ('ethereum', 'WBTC', 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599, 5),
+             ('ethereum', 'PEPE', 0x6982508145454ce325ddbe47a25d4ec3d2311933, 6),
```

Comment:
Let's make these lists a bit smaller to maybe 5 coins and not include meme coins and less known coins. We can add more later if needed.

Reply:
I will remake the list to shrink the size a bit, and easier for manually editing.